.github/workflows/run_main_tests.yml

```diff
@@ -22,9 +22,15 @@ jobs:
       run: | 
         python -m pip install -e .[dev]
     - name: Run linter checks
-      run: flake8 . && interrogate --verbose .
+      run: |
+        echo "=== Running style checks with flake8 ==="
```

Comment:
Can you ask Jon if he can add these prints to the template?

Reply:
Thanks @arielleleon for the review! Agreed that these print statements would be useful in the template and other repos. I was motivated to add this because I was confused about the coverage report generated by the `interrogate --verbose .` I interpreted it as a test coverage report, when it was actually a docstring coverage report. So I spent quite a long time (more than I want to admit!) trying to understand why I had 100% test coverage locally but couldn't get past this check. 

@jtyoung84 , tagging you for visibility. I'm about to request review from you also.